TACC/jupyterhub-admin
# task/SCIN-43 - jupyterhub users

JupyterHub users interface

Testing:
- Configure .env for the admin token
- Make sure your jupyterhub tenant metadata has the services configuration (described in README.md)
- Log in to jupyterhub, start a server for your user
- Run `make dev` on this repo
- Browse to localhost:8000
- Browse to your user, hit the stop server button
- Check your JupyterHub and see that the server has stopped
